packages/language-server/src/browser.ts

```
@@ -0,0 +1,19 @@
+// `vscode-languageserver/node` connection, the jsdom-backed HTML/a11y plugin).
+export { createService } from "./service/create-service";
+export { default as MarkoPlugin } from "./service/marko";
+export { default as ScriptPlugin } from "./service/script";
+export { default as StylePlugin } from "./service/style";
+export type { Plugin } from "./service/types";
+
+export { clearMarkoCacheForFile } from "./utils/file";
+export { default as setupMessages } from "./utils/messages";
+export * as documents from "./utils/text-documents";
+export { setSystem } from "./utils/ts-system";
+export * as workspace from "./utils/workspace";
```

ESLint

[error] Run autofix to sort these exports!

(simple-import-sort/exports)